src/main/java/agh/ddd/groups/group/commands/AddMembersCommand.java

```diff
@@ -0,0 +1,24 @@
+package agh.ddd.groups.group.commands;
+
+import agh.ddd.groups.group.Member;
+import agh.ddd.groups.group.valueobjects.GroupId;
+
+import java.util.Set;
+
+public class AddMembersCommand {
+    private GroupId id;
+    private Set<Member> members;
+
+    public AddMembersCommand(GroupId id, Set<Member> members) {
+        this.id = id;
+        this.members = members;
+    }
+
+    public GroupId getId() {
+        return id;
+    }
+
+    public Set<Member> getMembers() {
+        return members;
```

Comment:
unmodifiableSet albo copyof albo jeszcze coś innego, brakuje final

Generalnie to eventy i commandy leciałbym standardowo z equalsem, hashcodem i toStringiem (value objects)